spec/models/like_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Validations For the Like model' do
+    before(:each) do
+      @like = Like.new(author_id: 2, post_id: 7)
+    end
+
+    before { @like.save }
+
```

Comment:
- [ ] Kindly use some positive test cases, for the `like` model test.

Reply:
Hey @Deepakdanger please show me an example of how to implement positive test cases?

Reply:
Thanks @Deepakdanger